src/containerapp/azext_containerapp/_help.py

```diff
@@ -892,6 +892,18 @@
                                     "cloud=AzurePublicCloud" \\
                                     "queueLength": "5" "queueName": "foo" \\
               --scale-rule-auth "connection=my-connection-string-secret-name"
+    - name: Create a container app with a custom scale rule using identity to authenticate
+      text: |
+          az containerapp create -n my-containerapp -g MyResourceGroup \\
+              --image my-queue-processor --environment MyContainerappEnv \\
+              --user-assigned myUserIdentityResourceId --min-replicas 4 --max-replicas 8 \\
+              --scale-rule-name queue-based-autoscaling \\
+              --scale-rule-type azure-queue \\
+              --scale-rule-metadata "accountName=mystorageaccountname" \\
+                                    "cloud=AzurePublicCloud" \\
+                                    "queueLength": "5" "queueName": "foo" \\
```

Comment:
please check if the example of this param is correct or not.

Reply:
updated